package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "joyent-repos",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

Comment:
nit: 2.4.0, this is a feature addition rather thana bug fix.

Reply:
Will do.